### DISCUSSION TITLE
Are we 0.19 yet

### DISCUSSION THREAD

**Discussion by dmitry-timofeev** [jni-rs/jni-rs#295] [General]
Hi @maurolacy ,

Big thanks for leading the efforts around improving and extending AutoArray, and resolving the issues in the OpenJDK itself. Do you think it is ready to be released, or do you consider any breaking changes?

**Comment by maurolacy** [comment]
Hi @dmitry-timofeev,

First off, thanks to you and to @stanislav-tkach for all the great suggestions, reviews, and general support.

Thanks for asking. I think it's time for a new release. If there are breaking changes, they are well documented in the Changelog for users to deal with them.

**Reply by maurolacy** [reply]
I mean, yes, in my opinion it should be 0.19.0, as these changes are breaking.

**Reply by dmitry-timofeev** [reply]
Good, thanks! I've created #298

**Comment by rib** [comment]
Now that it's possible to mark discussions as closed (https://github.com/orgs/community/discussions/3097#discussioncomment-5316922) I think it makes sense that we close this discussion since the 0.19 release was finalized back in February 2021: https://github.com/jni-rs/jni-rs/pull/298
